client/lib/media/test/actions.js

```diff
@@ -69,6 +69,7 @@ describe( 'MediaActions', function() {
 						return {
 							get: mediaGet.bind( [ siteId, mediaId ].join() ),
 							update: mediaUpdate.bind( [ siteId, mediaId ].join() ),
+							edit: mediaUpdate.bind( [ siteId, mediaId ].join() ),
```

Comment:
Here we are mocking wpcom.media instance. https://github.com/Automattic/wp-calypso/pull/8945#issuecomment-256097274